### DISCUSSION TITLE
陸打造「AI檢察官」稱「準確度超過97％」

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#228] [科技新知]
AI檢察官很難實際執行，且讓AI進行決策和起訴感覺頗具爭議，若有判斷錯誤也很難釐清責任。
https://www.ettoday.net/dalemon/post/57912

**Comment by ChinChun0220** [comment]
我認為AI是能夠用來'輔助'案件偵查，處理瑣碎事情，但最終的審判應該還是要由專業人員進行，況且文中也沒提計97%數據是從何來(參考文章有鎖QQ)，統計能夠玩的數值花樣很多。

**Comment by luo910227** [comment]
我也認為AI不適合做為直接起訴的工具，因有時事件的複雜程度不是單純用AI學到的是非標準去判定就可以的，我認為如果他不可能97%準確率，如果還是挺有機會出了一般檢察官不會出的錯，那還不如不要使用。

**Comment by JAIMEHUNG** [comment]
覺得還是要由人類檢察官進行判決，但是ai可以做為輔助判決使用

**Comment by YuHengChen54** [comment]
AI還是有很多比人類不足的地方，感覺用在判決上不太好
